src/test/java/tr/com/srdc/cda2fhir/IntegrationTest.java

```diff
@@ -57,8 +69,47 @@ public static void init() throws IOException {
 	public void rakiaIntegration() throws Exception {
 		String sourceName = "Cerner/Person-RAKIA_TEST_DOC00001 (1).XML";
 		// create transaction bundle from ccda bundle
+
```

Comment:
This test is specifically checking the Rakia test file. You may want to make a new test if you're adding a bunch of elements. You can use the ClinicalDocumentMetataDataGenerator to create a document, and then add the org, address, agent, etc. to that. I actually recommend maybe making generators for each of those to maintain with the current pattern.

Reply:
Moved.